plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/logcat/LogcatClassVisitor.kt

```diff
@@ -0,0 +1,55 @@
+package io.sentry.android.gradle.instrumentation.logcat
+
+import org.objectweb.asm.ClassVisitor
+import org.objectweb.asm.MethodVisitor
+import org.objectweb.asm.Opcodes
+import org.objectweb.asm.commons.AdviceAdapter
+
+class LogcatClassVisitor(
```

Comment:
we could utilise `CommonClassVisitor` instead, because it does automatic logging and catching exceptions during visiting, but up to you, this is also fine as it is now. You could look at other instrumentables for examples